common/configuration/class.Component.php

```diff
@@ -134,13 +148,16 @@ public function getName()
      * Short description of method setName
      *
      * @access public
+     *
      * @author Jerome Bogaerts, <jerome.bogaerts@tudor.lu>
+     *
      * @param  string name
+     * @param mixed $name
```

Comment:
The annotation for this param is duplicated, I'd change the previous line from `string name` to `string $name` instead

```suggestion
```